docs/detections/rules-ui-create.asciidoc

```diff
@@ -157,9 +157,20 @@ When you use a saved query, the *Load saved query "_query name_" dynamically on
 
 * Deselect this to load the saved query as a one-time way of populating the rule's *Custom query* field and filters. This copies the settings from the saved query to the rule, so you can then further adjust the rule's query and filters as needed. If the saved query is later changed, the rule will not inherit those changes.
 
-. preview:[] (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Use *Suppress Alerts By* to reduce the number of repeated or duplicate alerts created each time the rule runs. Enter a field name to group matching source events by the field's unique values, and only one alert will be created for each group of events per rule execution. You can also enter multiple fields to group events by unique combinations of values.
+. preview:[] (Optional, https://www.elastic.co/pricing[Platinum or higher subscription] required) Use *Suppress alerts by* to reduce the number of repeated or duplicate alerts created by the rule.
+
+.. Enter a field name to group matching source events by the field's unique values; only one alert will be created for each group of events. You can also enter multiple fields to group events by unique combinations of values.
+
+.. Select how often to create alerts for duplicate events:
++
+--
+* *Per rule execution*: Create an alert each time the rule runs and matches duplicate events.
+* *Per time period*: Create one alert for all matching duplicate events within a specified time window, beginning when the rule first matches an event and creates the alert.
++
+For example, if a rule runs every 5 minutes but you don't need alerts that frequently, you can set the suppression time period to a longer time, such as 1 hour. If the rule matches an event, it creates an alert at that time, and for the next hour any subsequent matches will be suppressed.
+--
 +
-You'll still be able to view the original events associated with suppressed alerts by investigating the alert in Timeline. Refer to <<alert-suppression>> for more information.
+With either option, you can view the original events associated with suppressed alerts by investigating the alert in Timeline. Refer to <<alert-suppression>> for more information.
```

Comment:
```suggestion
With either option, you can view all events associated with suppressed alerts by investigating the alert in Timeline. Refer to <<alert-suppression>> for more information.
```
Just another option. Calling them original seems a bit strange since the events haven't changed (they just never generated individual alerts)